tables/audisp/src/fileeventstableplugin.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
```

Comment:
Can you do an audit of missing includes so this header is standalone? We seem to be missing e.g., `memory`, and `string`.

Reply:
The memory and string headers are already imported by the header that defines the table plugin type used as the base class: https://github.com/zeek/zeek-agent/blob/master/components/zeekdatabase/include/zeek/ivirtualtable.h#L4

Reply:
Marking as resolve as header files are already included.

Reply:
While you are right at this moment, I'd argue for self-contained headers. https://google.github.io/styleguide/cppguide.html#Self_contained_Headers collects some arguments (I agree with it on that particular point).